.circleci/config/@build-release.yml.back

```diff
@@ -0,0 +1,2049 @@
+version: "2.1"
```

Comment:
Did you intend to commit this file? There are a handful of unresolved merge conflicts in it

Reply:
ooops, nope!